### PR TITLE
fix: allow setting commands to null

### PR DESCRIPTION
**Pull request by 0utplay**
### Motivation
Currently there is no way to set null values in tasks using a command.

### Modification
Created some unset commands so that you can set null values using a command.
### Result
Null values can be set using commands.

### CODE REVIEW

File: node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java [hunk 1]
```diff
@@ -761,6 +761,48 @@ public void clearProcessParameter(
...
+    }
+  }
+
+  @CommandMethod("tasks task <name> unset runtime")
+  public void unsetRuntime(
+    @NonNull CommandSource source,
+    @NonNull @Argument("name") Collection<ServiceTask> serviceTasks
+  ) {
+    for (var serviceTask : serviceTasks) {
+      this.updateTask(serviceTask, builder -> builder.runtime(null));
```
Comment (derklaro) [inline node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java]: which runtime is used when we set it to null?

File: node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java [hunk 2]
```diff
@@ -761,6 +761,48 @@ public void clearProcessParameter(
...
   }
 
+  @CommandMethod("tasks task <name> unset javaCommand")
+  public void unsetJavaCommand(
+    @NonNull CommandSource source,
+    @NonNull @Argument("name") Collection<ServiceTask> serviceTasks
+  ) {
+    for (var serviceTask : serviceTasks) {
+      this.updateTask(serviceTask, builder -> builder.javaCommand(null));
+      source.sendMessage(I18n.trans("command-tasks-set-property-success",
```
Comment (schucko1337) [inline node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java:771]: I dont like this playo. This looks like a lot of duplicate code :( but if klaro likes it then it's okay.

Comment (0utplay) [reply node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java:771]: How would you like to do it?

File: node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java (hunk 1 above)
Comment (derklaro) [reply node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java]: this is not resolved

File: node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java (hunk 2 above)
Comment (derklaro) [reply node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java:771]: I think it's about moving the for loop into a seperate method, and I can see the issue. Probably best to move into another pull request tho? @robertschuck

**Review by derklaro — APPROVED** [review]

**Pull request merged**